src/Actions/index.tsx

```diff
@@ -25,6 +25,10 @@ export default class Actions extends React.Component<Props, State> {
   constructor (props: Props) {
     super(props)
 
+    this.state = {
```

Comment:
```suggestion
    this.state: State = {
```

Reply:
Otherwise, `actions.state` will be of type `any`

Reply:
It's a class property, not a variable, you can't set annotations like that. `this.state` is typed from the `React.Component<Props & DefaultProps, State>` line.